docs/getstarted.md

```diff
@@ -1,110 +1,169 @@
 (getstarted-page)=
 
-# Getting started
+# Get started
+
+If you're new to Nextflow, [take a look at the tutorials on this page](https://www.nextflow.io/blog/2023/learn-nextflow-in-2023.html#nf-core) first. This guide will help you get started with using Nextflow. It covers: 
+
+- Requirements
+- Installation
+- Your first script
+
+We also recommend taking a look at these pages once you've installed Nextflow.
+
+- {ref}`config-page`
+- {ref}`workflow-page`
+- {ref}`operator-page`
 
 (getstarted-requirement)=
 
 ## Requirements
 
-Nextflow can be used on any POSIX compatible system (Linux, macOS, etc). It requires Bash 3.2 (or later) and [Java 11 (or later, up to 21)](http://www.oracle.com/technetwork/java/javase/downloads/index.html) to be installed.
+Nextflow can be used on any POSIX-compatible system (Linux, macOS, etc). It requires Bash 3.2 (or later) and [Java 11 (or later, up to 21)](http://www.oracle.com/technetwork/java/javase/downloads/index.html) to be installed. You can check what version you have using the following command:
+
+```bash
+java -version
+```
 
-For the execution in a cluster of computers, the use of a shared file system is required to allow the sharing of tasks input/output files.
+We recommend that you install Java through [SDKMAN!](https://sdkman.io/), and that you use the latest LTS version of Temurin. See [this website](https://whichjdk.com/) for more information.
 
-Nextflow can also be run on Windows through [WSL](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux).
+To install Java with SDKMAN:
 
-:::{tip}
-We recommend that you install Java through [SDKMAN!](https://sdkman.io/), and that you use the latest LTS version of Corretto or Temurin. See [this website](https://whichjdk.com/) for more information. While other Java distros may work at first or even most of the time, many users have experienced issues that are difficult to debug and are usually resolved by using one of the recommended distros.
+1. Install SDKMAN:
+
+    ```bash
+    curl -s https://get.sdkman.io | bash
+    ```
+
+2. Open a new terminal and install Java:
 
-To install Corretto 17:
+    ```bash
+    sdk install java 17.0.10-tem
+    ```
+
+3. Confirm that Java is installed correctly:
+
+    ```bash
+    java -version
+    ```
+
+4. To install Temurin 17:
+
+   ```bash
+   sdk install java 17.0.6-tem
+   ```
+
+## Updates
+
+With Nextflow installed in your environment, you can update to the latest version using the following command:
```

Comment:
It explains how to install Java to update nextflow, without explaining how to install nextflow itself. It looks odds to me

Reply:
The sections are just out of order, I will fix it